README.md

```diff
@@ -62,16 +62,23 @@ android {
 ```
 
 ## Starting the PhotoEditor activity
-Simply add this line to the place from where you want to start the activity-
 
+In the activity from where you want to start the photo editor add this line outside all methods in your activity class-
 ```java
+final int PHOTO_EDITOR_REQUEST_CODE = 231;// Any integer value as a request code.
```

Comment:
`private static final int PHOTO_EDITOR_REQUEST_CODE = 231;`

Reply:
Here static is not required, since we're accessing the same variable in the same class. private is fine.